src/app/app-extras.module.ts

```diff
@@ -13,13 +13,20 @@ import {
   SkyTimepickerModule
 } from './public';
 
+import {
+  SkyFuzzyDateFactory
+} from './public/modules/datepicker/fuzzy-date-factory';
+
 @NgModule({
   exports: [
     SkyAppLinkModule,
     SkyDatePipeModule,
     SkyDatepickerModule,
     SkyTimepickerModule,
     SkyDateRangePickerModule
+  ],
+  providers: [
+    SkyFuzzyDateFactory
```

Comment:
This is already provided by the `SkyDatetimeModule`, yes? I think we can remove it from app extras.

Reply:
Removed